var/spack/repos/builtin/packages/py-parsl/package.py

```diff
@@ -18,6 +18,12 @@ class PyParsl(PythonPackage):
 
     version("1.1.0", sha256="6a623d3550329f028775950d23a2cafcb0f82b199f15940180410604aa5d102c")
 
+    variant(
+        "monitoring",
+        default=False,
+        description="enable live monitoring <https://parsl.readthedocs.io/en/stable/userguide/monitoring.html>",
```

Comment:
I don't think we want a URL in the description, `spack info py-parsl` will look weird. You can keep a comment in the recipe though.